libraries/AP_Beacon/AP_Beacon.h

```diff
@@ -116,4 +123,8 @@ class AP_Beacon
     // individual beacon data
     uint8_t num_beacons = 0;
     BeaconState beacon_state[AP_BEACON_MAX_BEACONS];
+
+    // fence boundary
+    Vector2f *_boundary = nullptr;
```

Comment:
don't need to set to nullptr, same with num_beacons. Everything in C++ is set to 0/nullptr when constructed. Explicitly setting it wastes program space